pkgs/by-name/op/openproject-cli/package.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  go_1_25,
+}:
+
+buildGoModule rec {
+  pname = "openproject-cli";
+  version = "0.5.3";
+
+  src = fetchFromGitHub {
+    owner = "opf";
+    repo = "openproject-cli";
+    rev = version;
+    hash = "sha256-5dovK9A2l+bT6hLGw8gyciK1p0ucGmdEX3qdC9zvJR4=";
+  };
+
+  vendorHash = "sha256-HiS8yHb8o2xdsCDKmebhJOCZpG/3rmE3zKcoaMYTxTE=";
+
+  # specify Go version explicitly
+  go = go_1_25;
+
+  subPackages = [ "." ];
+
+  ldflags = [
+    "-s"
+    "-w"
+  ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    # Create the canonical alias
+    ln -s "$out/bin/openproject-cli" "$out/bin/op"
+
+    # Set a writable HOME to avoid /homeless-shelter errors
+    export HOME="$PWD/tmp-home"
+    mkdir -p "$HOME"
```

This may not be spelled correctly
```suggestion
  nativeBuildInputs = [ installShellFiles writableTmpDirAsHomeHook ];

  postInstall = ''
    # Create the canonical alias
    ln -s "$out/bin/openproject-cli" "$out/bin/op"
```